OpenConsole.sln

```diff
@@ -228,6 +228,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Buffer", "Buffer", "{1E4A06
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared", "Shared", "{89CDCC5C-9F53-4054-97A4-639D99F169CD}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Types.Test.Unit", "src\types\ut\Types.Unit.Test.vcxproj", "{34DE34D3-1CD6-4EE3-8BD9-A26B5B27EC73}"

```

Comment:
This is now a third pattern of testing folders. 

We used to have:
/src/host/ut_host
/src/host/ft_host
/src/host/ft_uia

And then @carlos-zamora added something like
/src/cascadia/UnitTests_CascadiaCore

And now you're adding
/src/types/ut/.

Three different patterns.

I think we need to decide on one of these and make all the tests follow that pattern. I'm not saying pattern 1 was the best (to the contrary, it's probably not great). But we shouldn't have three different patterns.

Reply:
moved to ut_types to fit with the host model. later: updating UT_CascadiaCore